03_ocp_repo_sync.sh

```diff
@@ -60,3 +60,9 @@ popd
 
 # Install baremetal-operator
 sync_go_repo_and_patch github.com/metalkube/baremetal-operator https://github.com/metalkube/baremetal-operator.git
+
+# Update project name to openshift-machine-api
+pushd "${GOPATH}/src/github.com/metalkube/baremetal-operator/deploy"
+echo "  namespace: openshift-machine-api" >> service_account.yaml
```

Comment:
What do you think of copying the upstream files into a dir under dev-scripts and modify them there during 08_deploy_bmo.sh?

Reply:
+1 that would probably be cleaner than modifying the git checkout

Reply:
Yup, I can do that. I wasn't sure which approach would be preferable.